app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateWalletFragment.kt

```diff
@@ -89,19 +88,30 @@ class CreateWalletActivity : BaseActivity() {
     @JvmField
     var whiteColor = 0
 
-    override val contentViewId = R.layout.activity_create_wallet
-
     private val uiHandler = Handler()
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
+    override val contentViewId = R.layout.fragment_create_wallet
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
         init()
     }
 
+    private fun init() {
+        rootView.post { startViewAnimation() }
```

Comment:
Do you need to do a "rootView.post" here? I think onViewCreated is called by the UI thread, so you can just call the method I think. Next two lines are also UI updates, if they work, this line should work without post too.